migrations/v4.js

```diff
@@ -0,0 +1,30 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
+
+describe('adapt-contrib-blank - v2.0.0 > v4.1.3', async () => {
```

Comment:
Change occurred from `v4.1.2` to `v4.1.3` so suggest amend each instance of `v2.0.0` to `v4.1.2` to match migration script convention.

Reply:
Adjusted.